docs/index.rst

```diff
@@ -2,12 +2,12 @@
 Specreduce documentation
 ########################
 
-The `specreduce` package aims to provide a data reduction toolkit for optical
+The ``specreduce`` package aims to provide a data reduction toolkit for optical
 and infrared spectroscopy, on which applications such as pipeline processes for
 specific instruments can be built. The scope of its functionality is limited to
 basic spectroscopic reduction, with basic *image* processing steps (such as
-bias subtraction) instead covered by `ccdproc` and other packages, data
-analysis by `specutils` and visualization by `specviz` or `cubeviz`. A few
+bias subtraction) instead covered by ``ccdproc`` and other packages, data
```

Comment:
Should link to upstream package docs if possible.

Reply:
done.